packages/integration-tests/tests/__snapshots__/flat-config-types.test.ts.snap

```diff
@@ -16,7 +16,7 @@ exports[`flat-config-types eslint should work successfully 1`] = `
         "line": 38,
         "message": "'_otherCases' is defined but never used.",
         "messageId": "unusedVar",
-        "nodeType": null,
+        "nodeType": "Identifier",
```

Comment:
this should still be `null` based on how the rule works?

Reply:
Ah, might be a merge thing - merging everything in...